backend/tests/Notifo.Domain.Tests/Integrations/Discord/DiscordTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Notifo.Domain.Integrations.Discord;
+public sealed class DiscordTests
```

Comment:
Does it make sense to have a custom base class as we have for the other providers?

Reply:
I don't think so since this is more likely an integration test that tests specific functionality related to my implementation. E.g. some other implementations from Messaging don't support images, links etc.